.gitignore

```diff
@@ -8,4 +8,3 @@ coverage
 .env
 npm-debug.log
 .DS_Store
-dist
```

Comment:
@suppermancool I guess this has been committed accidentally.

Reply:
@maxceem done